src/main/webapp/WEB-INF/view/order/admin-orders.jsp

```diff
@@ -0,0 +1,25 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
```

Comment:
Should this reflect what we show on the page? For example, "Orders"? :)

Reply:
done